conf/locale/he/LC_MESSAGES/django.po

```diff
@@ -14134,6 +14140,11 @@ msgstr "הוסף את {course_name} לעגלה <span>({price} דולר ארה\"
 msgid "Enroll in {course_name}"
 msgstr "הירשם ב-{course_name}"
 
+#: lms/templates/courseware/course_about.html
+msgctxt "self"
+msgid "Enroll in {course_name}"
+msgstr "רישום אוטומטי ב-{course_name}"
+
```

Comment:
I add this only in .po but not in template due to we don't have the translation string and is not included in technical doc

Reply:
What exactly do you mean? does the string exist at the `lms/templates/courseware/course_about.html` template?

Reply:
Yes, the string ´Enroll in {course_id}....` exists in the template. But what I mean is what exactly would be the translation to Hebrew for "Enroll in"?

Reply:
Naturally, we are not expecting you to know hebrew. This will need to be handled by the translator from campus. Now, about adding the string `"in"` that is actually not a good idea. Because then you are assuming that the translation in every language works the same, which is not true. Other languages could work differently and use other preposition. Imagine that the literal translation for hebrew could be "enroll at" instead of "enroll in". That is why you should always extract the complete string